packages/workbox-routing/src/lib/router.js

```diff
@@ -32,6 +32,9 @@ import normalizeHandler from './normalize-handler';
  * If you'd like this to be handled automatically for you, calling
  * `addFetchListener()` will cause the `Router` to respond to `fetch` events.
  *
+ * If a request matches multiple routes, precedence will be given to the 
```

Comment:
@jeffposnick is this intended behavior? Seems like the opposite of what I'd expect.

Reply:
It is the intended behavior, yes: https://github.com/GoogleChrome/workbox/blob/master/packages/workbox-routing/src/lib/router.js#L294

I can't find the relevant discussion that preceded that code, but I assume we discussed it at the time 😄 

We could change it in the next major release if folks would prefer the opposite behavior.